themes/default/content/docs/concepts/options/_index.md

```diff
@@ -29,4 +29,5 @@ All resource constructors accept an options argument that provide the following
 - [replaceOnChanges](/docs/concepts/options/replaceonchanges/): declare that changes to certain properties should be treated as forcing a replacement.
 - [retainOnDelete](/docs/concepts/options/retainondelete/): if true the resource will be retained in the backing cloud provider during a Pulumi delete operation.
 - [transformations](/docs/concepts/options/transformations/): dynamically transform a resource’s properties on the fly.
```

Comment:
Do we need a comment at this level to help folks understand which of these to use?

Something like this?

```suggestion
- [transformations](/docs/concepts/options/transformations/): dynamically transform a resource’s properties on the fly.  Prefer `transforms` if possible. `transformations` will be deprecated in the future in favor of `transforms`. 
```